cherryontech/JourneyWise
# connected the hero button & the storytelling button to the modal

## Pull Request

### Description
I connected the hero section of the button and the story telling buttons to the security modal. 

### Related Issue
https://marisolsaumi.atlassian.net/browse/JW-44

### Description of Changes
* connected the  hereo section button  & the story telling button to show the security modal on click. 
* What problem does it solve or what feature does it add?

### Checklist
- [ x] I have read the CONTRIBUTING.md document.
- [x ] My code follows the code style of this project.
- [x ] I have updated the documentation or comments where necessary.
- [ x] All tests pass locally.

### Testing Instructions
pull the branch and run it on the local machine and click the buttons to ensure it is working properly

### Screenshots (if applicable)
[Add screenshots to help explain your changes, if applicable.]

### Gotchas/Learnings
I created a hook and that was a learning process. 

### Additional Notes (Optional)
* Include any other relevant information for reviewers.
